certbot-nginx/certbot_nginx/_internal/configurator.py

```diff
@@ -723,41 +724,62 @@ def _make_server_ssl(self, vhost: obj.VirtualHost) -> None:
 
         """
         https_port = self.config.https_port
-        ipv6info = self.ipv6_info(str(https_port))
-        ipv6_block = ['']
-        ipv4_block = ['']
+        http_port = self.config.http01_port
+
+        # no addresses should have ssl turned on here
+        assert not vhost.ssl
+
+        sslified_addrs: List[obj.Addr] = [obj.Addr(addr.get_addr(), str(https_port), True, False,
```

Comment:
very minor nit: maybe rename to something like `addrs_needing_ssl`? the variable name sorta implies these already have ssl

Reply:
good point!

Reply:
ok no now I remember why I did it that way -- the addrs in this list have in fact been sslified, they just haven't yet been added as listen blocks. the addr objects themselves are now ssl.

Reply:
not that it super matters whether these addrs have ssl in them since I just manually insert the blocks as having ssl and ignore the ssl property

Reply:
maybe `addrs_to_listen_on` though that's long. `addrs_to_insert`?